src/plugins/intel_cpu/tests/unit/transformations/arm/convert_conv_bias.cpp

```diff
@@ -0,0 +1,136 @@
+    std::shared_ptr<ov::op::v0::Parameter> input;
+    auto conv = createConvolution(input_type, weights_type, input);
+
+    auto bias = ov::op::v0::Constant::create(bias_type, {1, 16, 1, 1}, {1.5f});
+    auto add = createAdd(conv, bias, ov::element::f32, bias_type);
+
+    auto dequant_scale = ov::op::v0::Constant::create(ov::element::f32, {1}, {0.5f});
+    auto multiply = std::make_shared<ov::op::v1::Multiply>(add, dequant_scale);
+
+    return std::make_shared<ov::Model>(ov::OutputVector{multiply}, ov::ParameterVector{input});
+}
+
+// Pattern: Input -> Convolution -> Add(Round(bias)->Convert(i32)) -> Multiply -> Result
+static std::shared_ptr<ov::Model> createRefGraph(ov::element::Type input_type,
+                                                 ov::element::Type weights_type,
+                                                 ov::element::Type bias_type) {
+    std::shared_ptr<ov::op::v0::Parameter> input;
+    auto conv = createConvolution(input_type, weights_type, input);
+
+    auto bias = ov::op::v0::Constant::create(bias_type, {1, 16, 1, 1}, {1.5f});
+    auto round = std::make_shared<ov::op::v5::Round>(bias, ov::op::v5::Round::RoundMode::HALF_TO_EVEN);
+    auto convert = std::make_shared<ov::op::v0::Convert>(round, ov::element::i32);
+
+    // The transformation creates a TypeRelaxed Add when the original Add is not TypeRelaxed
+    auto add = createAdd(conv, convert, ov::element::f32, ov::element::i32, true);
+
+    auto dequant_scale = ov::op::v0::Constant::create(ov::element::f32, {1}, {0.5f});
+    auto multiply = std::make_shared<ov::op::v1::Multiply>(add, dequant_scale);
+
+    return std::make_shared<ov::Model>(ov::OutputVector{multiply}, ov::ParameterVector{input});
+}
+
+// u8 input, i8 weights, f32 bias -> transformation should be applied
+TEST_F(TransformationTestsF, ConvertConvolutionBias_U8Input_I8Weights_Applied) {
+    model = createInitGraph(ov::element::u8, ov::element::i8, ov::element::f32);
+    manager.register_pass<ConvertConvolutionBias>();
+    model_ref = createRefGraph(ov::element::u8, ov::element::i8, ov::element::f32);
+}
+
+// u8 input, u8 weights, f32 bias -> transformation should be applied
+TEST_F(TransformationTestsF, ConvertConvolutionBias_U8Input_U8Weights_Applied) {
+    model = createInitGraph(ov::element::u8, ov::element::u8, ov::element::f32);
+    manager.register_pass<ConvertConvolutionBias>();
+    model_ref = createRefGraph(ov::element::u8, ov::element::u8, ov::element::f32);
+}
+
+// i8 input, i8 weights, f32 bias -> transformation should be applied
+TEST_F(TransformationTestsF, ConvertConvolutionBias_I8Input_I8Weights_Applied) {
+    model = createInitGraph(ov::element::i8, ov::element::i8, ov::element::f32);
+    manager.register_pass<ConvertConvolutionBias>();
+    model_ref = createRefGraph(ov::element::i8, ov::element::i8, ov::element::f32);
+}
+
+// bias already i32 -> transformation should NOT be applied
```

Comment:
If the transformation shouldn't be applied, we don't need to create `model_ref`: it will be automatically cloned from `model`. This actual for all negative test cases